syncdata-default.properties

```diff
@@ -438,3 +438,15 @@ mosip.role.admin.syncdata.getv2clientsettings=REGISTRATION_SUPERVISOR,REGISTRATI
 mosip.role.admin.syncdata.getclientsettingsentityIdentifier=REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,default
 mosip.role.admin.syncdata.getscriptsscriptName=REGISTRATION_SUPERVISOR,REGISTRATION_OFFICER,default
 
+logging.level.io.mosip=DEBUG
+logging.level.org.springframework.web.client.RestTemplate=DEBUG
+
+spring.mvc.pathmatch.matching-strategy=ANT_PATH_MATCHER
```

Comment:
this property is not required we have already added in application-default.properties

Reply:
Done